nautobot/docs/user-guide/administration/guides/request-profiling.md

```diff
@@ -0,0 +1,85 @@
+# Request Profiling
```

Comment:
This file probably should be added to  the table of contents in `mkdocs.yml`?

Reply:
It is listed under the admin guides section there, did I miss something else?

```diff
--- a/mkdocs.yml
+++ b/mkdocs.yml
@@ -310,6 +310,7 @@ nav:
         - Permissions: "user-guide/administration/guides/permissions.md"
         - Prometheus Metrics: "user-guide/administration/guides/prometheus-metrics.md"
         - Replicating Nautobot: "user-guide/administration/guides/replicating-nautobot.md"
+        - Request Profiling: "user-guide/administration/guides/request-profiling.md"
         - Django Storages with S3: "user-guide/administration/guides/s3-django-storage.md"
         - Task Queues: "user-guide/administration/guides/celery-queues.md"
     - Feature Guides:
```

Reply:
No, but apparently I did :-) Sorry for the noise.